providers/ms-teams/src/lib/msTeams.provider.ts

```diff
@@ -14,11 +14,22 @@ export class MsTeamsProvider implements IChatProvider {
   constructor(private config) {}
 
   async sendMessage(data: IChatOptions): Promise<ISendMessageSuccessResponse> {
-    const response = await this.axiosInstance.post(
-      data.webhookUrl,
-      JSON.parse(data.content)
-    );
-
+    let response
+    
+    try {
+      response = await this.axiosInstance.post(
```

Comment:
axios.post can also throw an error, so in the catch logic there is no certainty that the error is caused by the data.content not being a JSON object.

I would suggest the following:
```
let payload

try {
  payload = JSON.parse(data.content);
} catch (err) {
  payload = { text: data.content };
}

const response = await this.axiosInstance.post(data.webhookUrl, payload);
```

Reply:
Aggree with this approach, it will wrap the relevant section with a try catch. @ducdung8491 do you mind addressing this before we can merge?

Reply:
ok, i would have catch message card format at the time